packages/host/app/components/matrix/room.gts

```diff
@@ -43,7 +43,7 @@ export default class Room extends Component<Signature> {
       data-test-room={{this.room.name}}
       data-test-room-id={{this.room.roomId}}
     >
-      {{#if this.room.messages}}
+      {{#if (or (gt this.room.messages.length 0) (bool this.pendingMessage))}}
```

Comment:
generally it is better to just create a getter that has a more readable name than to combine a lot of helpers in a template. this example is more more easily understandable:
```ts
  private get hasMessagesOrPending() {
    return this.room.messages.length > 0 || this.pendingMessage;
  }
  .
  .
  {{#if this.hasMessagesOrPending}}
```